src/metabase/pulse/render/body.clj

```diff
@@ -834,13 +834,24 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(defn- apply-funnel-viz-settings
+  [rows {order-data :funnel.rows}]
+  (let [rows (vec rows)]
+    (->> (for [{:keys [enabled originalIndex name]} order-data]
+           (when enabled
+             (cond-> (get rows originalIndex)
+               name (assoc 0 name))))
```

Comment:
There seems to be no way to set a custom row name.
![image](https://user-images.githubusercontent.com/1937582/202686276-af2f16b9-5f61-48c6-9aab-e3b07f9c68a6.png)

Maybe we can just use whatever returns from `(get rows originalIndex)` and drop `cond->`?

What do you think?